healthcheck/__init__.py

```diff
@@ -2,7 +2,6 @@
 __version__ = '0.0.7'
 __url__ = 'https://github.com/yola/healthcheck'
 
-
-from healthcheck import (HealthChecker, HealthCheck, ListHealthCheck,
-                         DjangoDBsHealthCheck, FilesExistHealthCheck,
-                         FilesDontExistHealthCheck)
+from .checks import (DjangoDBsHealthCheck, FilesDontExistHealthCheck,
```

Comment:
F401 '.checks.ListHealthCheck' imported but unused
F401 '.checks.DjangoDBsHealthCheck' imported but unused
F401 '.checks.FilesExistHealthCheck' imported but unused
F401 '.checks.HealthChecker' imported but unused
F401 '.checks.HealthCheck' imported but unused
F401 '.checks.FilesDontExistHealthCheck' imported but unused
I101 Imported names are in the wrong order. Should be DjangoDBsHealthCheck, FilesDontExistHealthCheck, FilesExistHealthCheck, HealthCheck, HealthChecker, ListHealthCheck

Reply:
Use noqa?  And now you can do absolute import here.